packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
+## [3.8.1]
+* Added reference in [example documentation](example/README.md) about [writing golden tests for smart widgets with dependency injection](doc/golden_tests.md).
+
 ## [3.8.0]
-* Added functionality to exclude smart widgets dependency injection components during golden tests.
+* Added functionality to exclude smart widgets dependency injection components during golden tests. [Read the documentation.](doc/golden_tests.md)
```

Comment:
Same as above

Reply:
Updated the links and reorganized how we are including them. ✌🏼